### PR TITLE
Feature/team one

### PR DESCRIPTION
**Pull request by doherty8**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -39,4 +39,15 @@ const hollieFuncOne = () => {
 
 const RaysNewFunction = () => {
   console.log("New function");
-};
\ No newline at end of file
+};
+
```
Comment (mz106) [inline server.js:43]: Nice one craig

File: server.js [hunk 2]
```diff
@@ -39,4 +39,15 @@ const hollieFuncOne = () => {
...
+const craigNewFunction = () => {
+    console.log("Craig's new function");
+  };
+
+const allisterFuncOne = () => {
```
Comment (mz106) [inline server.js:48]: Where do this come from?

**Review by mz106 — APPROVED** [review]
LGTM

File: server.js (hunk 1 above)
Comment (doherty8) [reply server.js:43]: Thanks

File: server.js (hunk 2 above)
Comment (doherty8) [reply server.js:48]: God knows

Comment (CodeDamien1) [reply server.js:48]: well isnt this fun

**Pull request merged**